src/patches/vault-historic-data-refetch.ts

```diff
@@ -0,0 +1,112 @@
+import { ChartTimeFrame } from '@badger-dao/sdk';
+import { ethers } from 'ethers';
+
+import { getDataMapper, getVaultEntityId } from '../aws/dynamodb.utils';
+import { ChartDataBlob } from '../aws/models/chart-data-blob.model';
+import { HistoricVaultSnapshotModel } from '../aws/models/historic-vault-snapshot.model';
+import { VaultDefinitionModel } from '../aws/models/vault-definition.model';
+import { getSupportedChains } from '../chains/chains.utils';
+import { toChartDataKey } from '../charts/charts.utils';
+import { getEnvVar } from '../config/config.utils';
+import { VaultStrategy } from '../vaults/interfaces/vault-strategy.interface';
+import { getStrategyInfo } from '../vaults/vaults.utils';
+
+/**
+ * Run Patch
+ * npx ts-node -r dotenv/config src/patches/vault-historic-data-refetch.ts
+ *
+ * Target Patch
+ * STAGE=staging | prod
+ * VAULT="0x..."
+ *
+ * Requires DDB write access + MFA authentication to patch.
+ *
+ * Due IO errors in indexer, default vaules could be saved in DDB and corrapt data in charts
+ * This patch will update the charts with the correct data.
+ *
+ * Data to be updated:
+ * - `strategy` field
+ *
+ * https://github.com/Badger-Finance/badger-api/issues/1539
+ */
+
+async function vaultHistoricDataRefetch() {
+  console.log('Patch:VaultSnapshotRefetch - Started');
+
+  getEnvVar('STAGE');
```

Comment:
does this need to be stored or used?

Reply:
use more like protector, need more for ddb setup

Reply:
so no